src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -28,6 +46,7 @@ public Drivetrain(CommandXboxController controller) {
 		frontRight = new CANSparkMax(1, MotorType.kBrushless);
```

Comment:
created `frontLeftController` but still initialize `frontLeft` - the whole `MotorController` setup is uncessary